### PR TITLE
feat: dynamic step size for the data for graphs

### PR DESCRIPTION
**Pull request by pranshuchittora**

### CODE REVIEW

File: frontend/.eslintignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
-build
\ No newline at end of file
+build
+.eslintrc.js
```
Comment (pranshuchittora) [inline frontend/.eslintignore:3]: Getting this error, tried various workarounds but nothing worked
Screenshot 2022-03-29 at 10 53 28

Comment (palash-signoz) [reply frontend/.eslintignore:3]: can you please ignore `eslintrc` and put it in `.eslintignore` _(with Axolo)_

Comment (palash-signoz) [reply frontend/.eslintignore:3]: possibly this would work _(with Axolo)_

Comment (pranshuchittora) [reply frontend/.eslintignore:3]: Yes, already did. Added this comment to highlight why I did this :sweat_smile: _(with Axolo)_

Comment (palash-signoz) [reply frontend/.eslintignore:3]: 👍

**Comment by pranshuchittora** [comment]
> GetQueryResults

Done 👍🏼

File: frontend/.eslintignore (hunk 1 above)
Comment (palash-signoz) [inline frontend/.eslintignore:3]: please remove this one and can you please include this one in `tsconfig.json` as discussed

Comment (palash-signoz) [reply frontend/.eslintignore:3]: this will solve the issue

**Review by palash-signoz — CHANGES_REQUESTED** [review]

**Review by palash-signoz — APPROVED** [review]

**Pull request merged**